### PR TITLE
Added MosaicId and NamespaceId support to the SearchBox. Network native mosaic first on the list. Reduced the number of columns for the mobile view

### PR DESCRIPTION
**Pull request by OlegMakarenko**
Resolved: #291 
Resolved: #292 
Resolved: #252 
Add names of the columns to config which have to be displayed in mobile view
`src\config\list-pages.json`
```
{
    "pages": [
        {
            "path": "/blocks",
            "name": "blocks",
            "props": {
                "storeNamespace": "block",
                "title": "blocks",
                "hasInfoText": true,
                "mobileColumns": [
                    "height",
                    "age",
                    "transactions",
                    "fee",
                    "harvester"
                ]
            }
        }
}

### CODE REVIEW

File: src/store/ui.js [hunk 1]
```diff
@@ -173,6 +201,9 @@ export default {
   }
 }
 
+const isMosaicId = (str) =>
```
Comment (AnthonyLaw) [inline src/store/ui.js]: MosaicID and NamespaceID is same length actually. 
maybe we should rename it to `isMosaicOrNamespaceId`

**Comment by AnthonyLaw** [comment]
for this method `isMosaicId`, it's only able to identify the mosaicID or namespaceID.

I like to suggest, if we can add Dropwdown to filter search Type 
- Account (Address, PublicKey)
- Transaction
- Mosaic (mosaicID or mosaic name)
- Namespace (namespaceID or namespace name)

File: src/store/ui.js (hunk 1 above)
Comment (OlegMakarenko) [reply src/store/ui.js]: Yes, good proposal, thanks. Changed in last commit

**Comment by OlegMakarenko** [comment]
> for this method `isMosaicId`, it's only able to identify the mosaicID or namespaceID.
> 
> I like to suggest, if we can add Dropwdown to filter search Type
> 
> * Default (block height)
> * Account (Address, PublicKey)
> * Transaction
> * Mosaic (mosaicID or mosaic name)
> * Namespace (namespaceID or namespace name)
> 
> @evias @OlegMakarenko

What is the probability of a coincidence of MosaicId and NamespaceId?

**Pull request merged**